packages/react/cypress.json

```diff
@@ -0,0 +1,19 @@
+{
+  "projectId": "yxfokk",
+  "componentFolder": "src",
+  "env": {
+    "coverage": false,
```

Comment:
Do we want to open another issue to actually collect coverage and merge it with jest similar to this https://github.com/bahmutov/cypress-and-jest?

Reply:
yes I have it as a to do to open this issue.

Reply:
added to this PR